### ISSUE TITLE
libmpv: resize glitches when embedding video window into layer backed NSView

### ISSUE THREAD

**Issue by devnikor** [mpv-player/mpv#2873]
I've record a [demo](https://www.dropbox.com/s/anwjoktvenbqz92/glitch.mov?dl=0)
I using ffmpeg and mpv from HEAD

**Comment by wm4** [comment]
Yeah, native embedding mode can be glitchy. I'm not sure what exactly might be causing it, or if it's fixable in theory. The recommendation is using opengl-cb embedding instead.

**Comment by devnikor** [comment]
Thanks! I'll try this tomorrow, but it is stable enough?

**Comment by wm4** [comment]
It's certainly more stable than the window embedding method, but also requires more work in certain areas. I recommend reading the documentation and examples.

**Comment by devnikor** [comment]
Ok, thanks for suggesting!


27 февр. 2016 г., 1:44 +0300, V. Lang<notifications@github.com>, писал:

**Comment by Akemi** [comment]
since the video was taken down i am not sure which glitches were reported here. PR #4391 switches to a layer backed view and also fixes most of the glitches i encountered. so it should also fix this issue.

if you still can test it feel free to.
